src/api/spec/models/bs_request_spec.rb

```diff
@@ -68,4 +68,30 @@
       end
     end
   end
+
+  describe '#changestate' do
+    let(:request) { create(:bs_request) }
+    let(:admin) { create(:admin_user) }
+
+    context 'for wiping request' do
+      before do
+        request
+
+        # assign the admin after the first call of 'request'. this is important to avoid errors
+        User.current = admin
```

Comment:
`user` is not specified in that context. When I go with `request.creator`, then it would raise an error:
```
Deletion of a request is only permitted for administrators. Please revoke the request instead.
```

That's the reason why I used the `admin`.